xendit/models/ewallet/ewallet.py

```diff
@@ -237,3 +282,90 @@ def _parse_ewallet_type(ewallet_type):
             return ewallet_type.name
         except AttributeError:
             return ewallet_type
+
+    @staticmethod
+    def create_ewallet_charge(
+        *,
+        reference_id,
+        currency,
+        amount,
+        checkout_method,
+        channel_code=None,
+        channel_properties=None,
+        customer_id=None,
+        basket=None,
+        metadata=None,
+        for_user_id=None,
+        with_fee_rule=None,
+        **kwargs,
+    ):
+        """Send POST Request to create EWallet Charge (API Reference: eWallets/Create E-Wallet Charge)
+
+        Args:
+          - reference_id (str)
+          - currency (str)
+          - amount (int)
+          - checkout_method (str)
+          - channel_code (str)
+          - channel_properties (dict)
+          - customer_id (str)
+          - basket (list)
+          - metadata (dict)
+          - **for_user_id (str) (XenPlatforms only)
+          - **with_fee_rule (str) (XenPlatforms only)
+
+        Returns:
+          EWalletCharge
+
+        Raises:
+          XenditError
+
+        """
+        url = "/ewallets/charges"
+        headers, body = _extract_params(
+            locals(),
+            func_object=EWallet.create_ewallet_charge,
+            headers_params=["for_user_id", "with_fee_rule"],
+        )
+        kwargs["headers"] = headers
+        kwargs["body"] = body
+
+        resp = _APIRequestor.post(url, **kwargs)
+        if resp.status_code >= 200 and resp.status_code < 300:
+            return EWalletCharge(**resp.body)
+        else:
+            raise XenditError(resp)
+
+    @staticmethod
+    def get_ewallet_charge_status(
+        *,
+        charge_id,
+        for_user_id=None,
+        **kwargs,
+    ):
+        """Get ewallet charge status of given charge id (API Reference: eWallets/Get E-Wallet Charge Status)
+
+        Args:
+          - charge_id (str)
+          - **for_user_id (str) (XenPlatforms only)
```

Comment:
same comment like above
```suggestion
        *,
        charge_id: str,
        for_user_id: str =None,
        **kwargs,
    ):
        """Get ewallet charge status of given charge id (API Reference: eWallets/Get E-Wallet Charge Status)

        Args:
          - charge_id (str)
          - **for_user_id (str) (XenPlatforms only)
```